src/main/java/com/spotify/logging/LoggingConfigurator.java

```diff
@@ -458,4 +461,15 @@ private static int getSyslogPort() {
     return isNullOrEmpty(port) ? -1 : Integer.valueOf(port);
   }
 
+  private static String getHeliosHostname() {
+    for (Map.Entry<String, String> entry : getenv().entrySet()) {
```

Comment:
Can you use Java 8 streams here instead? A stream().filter().findAny().orElse(null) would be a good fit here.